suite-native/atoms/src/Sheet/useShakeAnimation.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    useAnimatedStyle,
+    useSharedValue,
+    withSequence,
+    withSpring,
+    withTiming,
+} from 'react-native-reanimated';
+
+const SHAKE_BY_DEGREES = 2;
+const SHAKE_DURATION = 100;
+const INITIAL_ROTATION = '0deg';
+
+export const useShakeAnimation = () => {
+    const rotation = useSharedValue(INITIAL_ROTATION);
+
+    const runShakeAnimation = () => {
+        'worklet';
```

Comment:
Do we need `worklet` here? Honestly I am not sure. Can you please double-check and give me brief explanantion here why you think it should/shoudn't be here?

Reply:
You are right. There is no need to use the `worklet` directive. Since `useAnimatedStyle` is used, the Reanimated library is handling it on its own. The worklet is created under the hood inside of the `useAnimatedStyle` hook, as described in this documentation [paragraph](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/worklets#using-hooks).

Reply:
fixed in 3f3466eb2

Reply:
This is not inside of useAnimatedStyle, it's used to fill sharedValue in `rotation` and that is used in `useAnimatedStyle` so I woudn't be so sure about that. 

I think it's because `withTiming` have worklet inside if you check implementation [in source](https://github.com/software-mansion/react-native-reanimated/blob/main/src/reanimated2/animation/timing.ts) so that means `worklet` could be still useful here, in case there will be any complicated calculations before withTIming/sequence itself (which is not this case).